packages/react-app-revamp/config/wagmi/custom-chains/evmosTestnet.ts

```diff
@@ -0,0 +1,19 @@
+export const evmosTestnet = {
+    id: 9001,
```

Comment:
Just wanted to check here - is the chainId of the testnet the same as the mainnet?